hipparchus-parent/pom.xml

```diff
@@ -491,6 +491,8 @@
     <hipparchus.hamcrest-library.version>1.3</hipparchus.hamcrest-library.version>
     <hipparchus.reflow-velocity-tools.version>1.1.1</hipparchus.reflow-velocity-tools.version>
     <hipparchus.velocity.version>1.7</hipparchus.velocity.version>
+    <!-- sonar related properties -->
+    <sonar.host.url>https://sonar.orekit.org/</sonar.host.url>
```

Comment:
I would just change the sonar.host.url property into hipparchus.sonar.host.url for the sake of consistency
with existing properties.

Reply:
If I am not mistaken, the name of the property is imposed by the Sonar plugin for Maven.

Reply:
I you prefer, I can pass the `sonar.host.url` property on the command line, through an environment variable:

```bash
mvn -B -f pom.xml sonar:sonar -Dsonar.login=$SONARQUBE_TOKEN -Dsonar.host.url=$SONARQUBE_HOST -Dsonar.branch.name=${GITHUB_REF##*/}
```

Reply:
If the plugin enforces the name, then we will use this name.